OpenRA.Mods.Common/Lint/CheckVoiceReferences.cs

```diff
@@ -0,0 +1,60 @@
+﻿#region Copyright & License Information
+/*
+ * Copyright 2007-2015 The OpenRA Developers (see AUTHORS)
+ * This file is part of OpenRA, which is free software. It is made
+ * available to you under the terms of the GNU General Public License
+ * as published by the Free Software Foundation. For more information,
+ * see COPYING.
+ */
+#endregion
+
+using System;
+using System.Linq;
+using System.Reflection;
+using OpenRA.Traits;
+
+namespace OpenRA.Mods.Common.Lint
+{
+	public class CheckVoiceReferences : ILintPass
+	{
+		public void Run(Action<string> emitError, Action<string> emitWarning, Map map)
+		{
+			foreach (var actorInfo in map.Rules.Actors)
+			{
+				var voiceSet = string.Empty;
```

Comment:
This here (actually the line that sets it below, but that is too overcrowded already) assumes the actor will have only one voice set, and will ignore all but the first if there are multiple. Shouldn't be too hard to remove the `.FirstOrDefault()` from [line 31](https://github.com/OpenRA/OpenRA/pull/8412/files#diff-53fde2db49577802d0410cae9f4d5c2dR31) and add a `foreach(var voiceSet in voiceSets)` around lines [44](https://github.com/OpenRA/OpenRA/pull/8412/files#diff-53fde2db49577802d0410cae9f4d5c2dR44) through 53 (after you rename this to `voiceSets`, of course).